docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -151,7 +146,7 @@ If you log in with Bitbucket, Codacy requires the following [permissions/scopes]
   <tbody>
     <tr>
       <td><code>account:write</code></td>
-      <td>Codacy retrieves the user's email addresses to enforce which commits are eligible for analysis. Furthermore, <a href="#why-does-codacy-ask-for-permission-to-create-ssh-keys">Codacy creates an SSH key</a> on the repository to allow cloning and integrating with your repository.</td>
+      <td>Codacy retrieves the user's email addresses to enforce which commits are eligible for analysis.</td>
```

Comment:
I'm removing SSH information from Bitbucket's `account:write` permission, as we are not currently using it.

Reply:
We should check with the development team if this permission is **effectively** not requested by Codacy while integrating with Bitbucket. :thinking:

Reply:
It also seems to [contradict this line](https://github.com/codacy/docs/pull/1765/files#diff-27cc3ef13af5641af71e6e8798c468f015af2409ab3f0306ba5092502c12e421R23). :thinking:

Reply:
@angelabarros @machadoit I removed this sentence from the Bitbucket `account:write` permission:
_"Furthermore, Codacy creates an SSH key on the repository to allow cloning and integrating with your repository."_

because 1) we already direct the user to the Bitbucket page to generate a new user key, and 2) now only admins can add repositories to Codacy (the fallback when the user has no admin permissions doesn't apply anymore).

However, I'm not sure if we still request this permission to Bitbucket users, as @prcr refers [above](https://github.com/codacy/docs/pull/1765/files#r1245074807).

Can you please validate if we can safely remove this sentence from the Bitbucket `account:write` permission? 🙏
We keep the sentence for `repository:admin` permission.

Reply:
@claudiacarpinteiro  We don't request this permission to Bitbucket users anymore. You can remove the sentence :)